pandas/core/accessors.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+
+An example/recipe for creating a custom accessor.
```

Comment:
Agree that docs should live in `internals.rts`, logical companion to the subclassing docs